frontend/src/pages/auth/styles.css

```diff
@@ -1,9 +1,9 @@
-.auth-container {
+.common-container {
     display: flex;
     min-height: 70vh;
 }
 
-.auth-content {
+.common-content {
```

Comment:
why?

Reply:
i reused it not only in auth pages, so i have to move it to `common/styles` i think

Reply:
but common name is too common. Name it more detailed. Not clear when to use it